src/simple_commander/main.py

```diff
@@ -220,21 +229,24 @@ def hit(self, other_unit):
         unit_class_name = other_unit. __class__.__name__
         logging.info('In hit - %s and %s' % (self.__class__.__name__, unit_class_name))
         if unit_class_name == 'Hero':
+            other_unit.hits += 1
```

Comment:
here you add hits directly to unit, but... in line #236 you call `self.controller.add_hits(other_unit)` - does it have any difference? so complicated...